plugins/SitesManager/lang/en.json

```diff
@@ -60,6 +60,7 @@
         "MainDescription": "Your Web Analytics reports need Websites! Add, update, delete Websites, and show the JavaScript to insert in your pages.",
         "MenuManage": "Manage",
         "MobileAppsAndSDKs": "Mobile apps and SDKs",
+        "MobileAppsAndSDKsModified": "Mobile Apps and SDKs",
```

Comment:
Only changing the writing case of `apps`, doesn't require to change the key.

Reply:
Thanks :+1: